### DISCUSSION TITLE
SslHandshakeException while sending an e-mail

### DISCUSSION THREAD

**Discussion by DotCat1985** [OrchardCMS/OrchardCore#10806] [Q&A] [answered]
Hi,
In my Web application developed with OrchardCore 1.0.0-rc2-13450, I configured SMTP for sending e-mail as follows:

**HOST NAME: mail5004.site4now.net
PORT NUMBER: 587
ENCRYPTION METHOD: STARTTLS
USERNAME: info@tokyodancemusic.com
PASSWORD: [password of info@tokyodancemusic.com]**

but I receive the error message below when I tried to send an e-mail (see the attached screenshots):

**An error occurred while sending an email: 'An error occurred while attempting to establish an SSL or TLS connection. This usually means that the SSL certificate presented by the server is not trusted by the system for one or more of the following reasons: 1. The server is using a self-signed certificate which cannot be verified. 2. The local system is missing a Root or Intermediate certificate needed to verify the server's certificate. 3. A Certificate Authority CRL server for one or more of the certificates in the chain is temporarily unavailable. 4. The certificate presented by the server is expired or invalid. It is also possible that the set of SSL/TLS protocols supported by the client and server do not match. See https://github.com/jstedfast/MailKit/blob/master/FAQ.md#SslHandshakeException for possible solutions. '**

How can I solve this issue? Thank you for attention.

**Comment by Skrypt** [comment, marked as answer]
You need to fix the SSL licence registration on the email server.

**Reply by DotCat1985** [reply]
Ok. Now I solved the issue fixing ssl licence registration through hosting provider staff.
Thank you for your support.
